src/Components/Home.js

```diff
@@ -0,0 +1,31 @@
+// Modules
+import { useEffect, useState } from "react";
```

Comment:
probably don't need useEffect/useState imported here since it's not used in this component?

Reply:
done!